core/imageroot/var/lib/nethserver/node/uninstall.sh

```diff
@@ -37,7 +37,8 @@ for modulehome in /var/lib/nethserver/*; do
     fi
     moduleid=$(basename $modulehome)
     echo "Deleting rootfull module ${moduleid}..."
-    systemctl disable --now eventsgw@${moduleid} agent@${moduleid} && rm -rf "${modulehome}"
+    systemctl disable --now eventsgw@${moduleid} agent@${moduleid} ${moduleid} && rm -rf "${modulehome}"
+    rm -f /etc/systemd/system/${moduleid}.service
```

Comment:
There can be more than one service started by a rootfull module. We could try to iterate over all units starting with ${moduleid}. For instance

    systemctl list-unit-files -q  -o json "${moduleid}*"  | jq -r .unit_file

Reply:
I tried to implement the cleanup in #185